src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IParenthesized.cs

```diff
@@ -0,0 +1,392 @@
+              Left: 
+                IParameterReferenceExpression: a (OperationKind.ParameterReferenceExpression, Type: System.Int32) (Syntax: 'a')
+              Right: 
+                IParameterReferenceExpression: b (OperationKind.ParameterReferenceExpression, Type: System.Int32) (Syntax: 'b')
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyOperationTreeAndDiagnosticsForTest<ParenthesizedExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void TestParenthesizedMultipleNesting04()
+        {
+            string source = @"
+class P
+{
+    static int M1(int a, int b)
+    {
+        return ((/*<bind>*/(a + b)/*</bind>*/));
+    }
+}
+";
+            string expectedOperationTree = @"
+IParenthesizedExpression (OperationKind.ParenthesizedExpression, Type: System.Int32) (Syntax: '(a + b)')
+  Operand: 
+    IBinaryOperatorExpression (BinaryOperatorKind.Add) (OperationKind.BinaryOperatorExpression, Type: System.Int32) (Syntax: 'a + b')
+      Left: 
+        IParameterReferenceExpression: a (OperationKind.ParameterReferenceExpression, Type: System.Int32) (Syntax: 'a')
+      Right: 
+        IParameterReferenceExpression: b (OperationKind.ParameterReferenceExpression, Type: System.Int32) (Syntax: 'b')
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyOperationTreeAndDiagnosticsForTest<ParenthesizedExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void TestParenthesizedMultipleNesting05()
+        {
+            string source = @"
+class P
+{
+    static int M1(int a, int b)
+    {
+        return (((/*<bind>*/a + b/*</bind>*/)));
+    }
+}
+";
+            string expectedOperationTree = @"
+IBinaryOperatorExpression (BinaryOperatorKind.Add) (OperationKind.BinaryOperatorExpression, Type: System.Int32) (Syntax: 'a + b')
+  Left: 
+    IParameterReferenceExpression: a (OperationKind.ParameterReferenceExpression, Type: System.Int32) (Syntax: 'a')
+  Right: 
+    IParameterReferenceExpression: b (OperationKind.ParameterReferenceExpression, Type: System.Int32) (Syntax: 'b')
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyOperationTreeAndDiagnosticsForTest<BinaryExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void TestParenthesizedConversion()
+        {
+            string source = @"
+class P
+{
+    static long M1(int a, int b)
+    {
+        return /*<bind>*/(a + b)/*</bind>*/;
+    }
+}
+";
+            string expectedOperationTree = @"
+IParenthesizedExpression (OperationKind.ParenthesizedExpression, Type: System.Int64) (Syntax: '(a + b)')
```

Comment:
Thanks, I fixed this issue, however I notice that conversion is incorrectly associated with syntax "a + b" instead of "(a + b)". I file a tracking issue to fix the conversion bind method.

---
In reply to: [147800460](https://github.com/dotnet/roslyn/pull/22846#discussion_r147800460) [](ancestors = 147800460)

Reply:
> I file a tracking issue to fix the conversion bind method

I am not sure if that would be the right way to address the issue. It might break SemanticModel behavior.

---
In reply to: [148059578](https://github.com/dotnet/roslyn/pull/22846#discussion_r148059578) [](ancestors = 148059578,147800460)